codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/HuffmanDecoder.java

```diff
@@ -213,4 +139,117 @@ private static void insert(Node root, int symbol, int code, byte length) {
             current.children[i] = terminal;
         }
     }
+
+    private static final class DecoderProcessor implements ByteProcessor {
+        private final int initialCapacity;
+        private byte[] bytes;
+        private int index;
+
+        private Node node = ROOT;
```

Comment:
no need to do the assignment here ... this should be the responsibility of `reset()` right?